.github/workflows/tests.yml

```diff
@@ -42,6 +42,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install tox
+        pip install -r requirements.txt
```

Comment:
Are these special requirements for the tests only?
Is there a linl generally to the failing tests?

Reply:
Hm. This stops it failing on Python3.4 but Python3.4 is not in the matrix.... Would this go to the 4.x branch?

Reply:
Link to failing tests https://travis-ci.org/github/collective/icalendar/jobs/774624957

Reply:
> Are these special requirements for the tests only?

I just added a generic `requirement.txt` file. 
`typing` module is a requirement for some tests in python3.4.